static/sass/styles.scss

```diff
@@ -130,7 +130,7 @@ $color-brand: #7c355d;
 }
 
 .p-media-object__image.is-small {
-  margin: 0 1.25rem 0 4px;
+  margin: 0 calc(1.25rem - 4px) 0 8px;
```

Comment:
@anthonydillon Apologies for the nitpicking! But these pixel values will only work up until [$breakpoint-x-large](https://github.com/canonical-web-and-design/vanilla-framework/blob/5ebc2f361551971028c80f207d450f1c5a64afa3/scss/_settings_breakpoints.scss#L8), where root font size is bumped up to 18px. On the other hand, rem values would work always and would make the line look simpler: ```margin: 0 1rem 0 .5rem```

Reply:
Sorry, rushing and jumping between projects. Done :+1: